pyproject.toml

```diff
@@ -41,7 +41,8 @@ dependencies = [
   "protobuf>=3.20, <6.0",
   "grpcio",
   "tqdm",
-  "requests",
+  "requests>=2.1, <3",
+  "requests-toolbelt>=1, <2",
```

Comment:
Can we start migrating away from `requests` and toward `httpx`? They have better async support and multipart file uploads built-in https://www.python-httpx.org/quickstart/#sending-multipart-file-uploads

Reply:
ah thx. i didn't know that